src/docs/metrics/mvrv/index.md

```diff
@@ -13,9 +15,6 @@ MVRV (Market Value to Realized Value) ratio is defined as an asset's market capi
 
 MVRV shows the ratio between the current price and the average price of every coin/token acquired. The more the ratio increases, the more people will be willing to sell as the potential profits increase. The value of MVRV gives an idea of how much overvalued or undervalued an asset is.
 
-### MVRV Indicator Explainer Video
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/vbYQLNJ40yw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
```

Comment:
You are changing from this:
<img width="825" alt="image" src="https://github.com/santiment/academy/assets/6518376/d0cfb943-043b-4b84-ae0a-6a1c120c2bde">
to this:
<img width="699" alt="image" src="https://github.com/santiment/academy/assets/6518376/5d844ff3-e7b0-42cb-be7e-6ee232840b3d">

Is this an expected change -- the `Definition` not to be the first thing on the page? For SEO purpose, as far as I understood, the metric name (and description?) should be as high as possible in the page.

Reply:
That's intended. The definition is still pretty high. I think we can use this format for every article in Academy - add a video on top which covers the article material. It won't work if the video is in a section, because the section would have to be in a different place on every article